Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/survey/SurveySettings.kt

```diff
@@ -42,7 +43,32 @@ class SurveySettings @Inject constructor(
                 .putString(KEY_OTP, if (value == null) null else gson.toJson(value))
                 .apply()
 
+    var otpAuthorizationResult: OTPAuthorizationResult?
```

Comment:
Because we now store the `AuthorizationResult` I think there is no reason to store the `otp` in the preferences anymore?

Reply:
yes, if you want to have a look at OTPReposititory.otpAuthorizationResult.set: the otp is removed from preferences.

Reply:
I mean we actually **never** have to store the otp in the preferences? Since the flow is always Generate OTP => Authorize OTP, the OTP would only be in the `preferences` for a couple of milliseconds or seconds. Or am I wrong?

Reply:
what if the connection breaks when trying to authorize?

Reply:
can we leave it for now? i see your point, but i'm afraid to delete already.